README.md

```diff
@@ -68,6 +68,12 @@ Silent installation of the latest version of the CLI can be achieved via the fol
 msiexec.exe /qn /i "$env:TEMP\NewRelicCLIInstaller.msi" | Out-Null; `
 ```
 
+#### Scoop
```

Comment:
Can we move this to the top of the Windows section?

Reply:
Sure thing

Reply:
Done